packages/nimble-components/src/table-column/tests/table-column.stories.ts

```diff
@@ -0,0 +1,416 @@
+import { html, ref, when } from '@microsoft/fast-element';
```

Comment:
What about moving these files to table-column\base\tests? I think that's what these stories represent, the base features shared across all columns.